graphene/sys/src/lib.rs

```diff
@@ -62,11 +62,6 @@ pub const GRAPHENE_RAY_INTERSECTION_KIND_ENTER: graphene_ray_intersection_kind_t
 pub const GRAPHENE_RAY_INTERSECTION_KIND_LEAVE: graphene_ray_intersection_kind_t = 2;
 
 // Constants
-pub const GRAPHENE_HAS_GCC: c_int = 1;
-pub const GRAPHENE_HAS_SCALAR: c_int = 1;
-pub const GRAPHENE_HAS_SSE: c_int = 1;
-pub const GRAPHENE_PI: c_double = 3.141593;
-pub const GRAPHENE_PI_2: c_double = 1.570796;
 pub const GRAPHENE_SIMD_S: *const c_char = b"sse\0" as *const u8 as *const c_char;
```

Comment:
This one should also be ignored while we're at it. It's completely dependent on where the graphene version was built that was used to generate the .gir file.

Reply:
Done